inst/tinytest/test_url_parsing.R

```diff
@@ -23,7 +25,24 @@ for(i in which(grepl("embl", names(paths))) ) {
   expect_equal(parsed$region,   "auto")
 }
 
+for(i in which(grepl("eopf_path_1", names(paths))) ) {
+  expect_silent(parsed <- .url_parse_other(paths[[ i ]]))
```

Comment:
tinytest, as opposed to testthat, doesn't rely on `pkg::load_all()` so internal functions have to be prefixed by `:::`:

```suggestion
  expect_silent(parsed <- Rarr:::.url_parse_other(paths[[ i ]]))
```

Reply:
Oh thank you! I've been having some trouble building the package locally so I was kind of doing "dummy" testing and forgot to add this back in.